frontend/src/components/MapChat/Message.js

```diff
@@ -11,25 +11,25 @@ function addZero(i) {
   return i
 }
 
-function formatDate(created_at) {
-  let date = new Date(created_at)
+function formatDate(createdAt) {
+  let date = new Date(createdAt)
   let formatted = (date.getMonth() + 1) + '/' + date.getDate()
   formatted += ' ' + addZero(date.getHours()) + ':' + addZero(date.getMinutes())
   return formatted
 }
 
 const Message = props => {
-  const { user_image, user_name, message, created_at, heading } = props
+  const { user_image: userImage, user_name: userName, message, created_at: createdAt, heading } = props
```

Comment:
haven't seen this before. this sets var `userImage` off the `user_image` key off props?